libs/dh/metering-point/feature-overview/src/lib/identity/em-dash.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const emDash = '—';
```

Comment:
question: this is used several places, including primary master data. Should we move this to a shared place? Or let it be and create another story where we fix this across the codebase?

Reply:
I've noticed that it's used in several places as well. My suggestion is to let it be for the identity and primary master data user stories and refactor it as part of the secondary master data because it's used a lot more there.

As for refactoring, we can create a custom structural directive which will automatically fallback to show the em dash in case a property is `false`/`undefined`/`null`. This way we avoid using `ngIf`s and/or ternary operators in the templates. What do you think about that?